autosklearn/ensembles/ensemble_selection.py

```diff
@@ -18,6 +18,7 @@ def __init__(
         bagging: bool = False,
         mode: str = 'fast',
         random_state: np.random.RandomState = None,
+        precision: int = 64,
```

Comment:
I don't think you need to expose this. It should be sufficient to pass the dtype to the array constructor.